R/whirl_r_session.R

```diff
@@ -230,6 +230,24 @@ wrs_check_status <- function(self, private, super) {
 wrs_log_script <- function(script, self, private, super) {
   private$current_script <- script
 
+  # Set the execute directory of the Quarto process calling the script
+  quarto_execute_dir <- options::opt("execute_dir", env = "whirl")
+  if (is.null(quarto_execute_dir)) {
+    quarto_execute_dir <- switch(
+      get_file_ext(script),
```

Comment:
I see this function "get_file_ext" everywhere in our packages but we have reintroduce tools::file_ext because it is a R base package

Reply:
I agree, it is a bit artificial that we this way have one less imported package.
Think I'll leave it for now, and then take a look before the first CRAN release, since we still need to remove more packages.